scopes/harmony/aspect-loader/is-esm-module.ts

```diff
@@ -0,0 +1,14 @@
+import findRoot from 'find-root';
+import { join } from 'path';
+import { readFileSync } from 'fs';
+
+export function isEsmModule(path: string) {
+  try {
+    const root = findRoot(path);
+    const packageJsonString = readFileSync(join(root, 'package.json')).toString('utf-8');
```

Comment:
Don't we have a better way? That's a lot of i/o and it's going to make loading aspects even longer. Can we introduce cache somewhere to speed things up?

Reply:
yes we should cache it for sure
but I don't think we really get here for aspects that were already loaded. as we just skip them by the id